src/referencePackages/src/ProjectsToBuild.props

```diff
@@ -256,5 +256,10 @@
     <ProjectsToBuild Include="$(ReferenceAssemblySourcePath)microsoft.build.utilities.core/16.6.0/Microsoft.Build.Utilities.Core.csproj"/>
     <ProjectsToBuild Include="$(ReferenceAssemblySourcePath)system.text.encoding.codepages/4.5.1/System.Text.Encoding.CodePages.csproj"/>
     <ProjectsToBuild Include="$(ReferenceAssemblySourcePath)microsoft.build/16.6.0/Microsoft.Build.csproj"/>
+    <ProjectsToBuild Include="$(ReferenceAssemblySourcePath)system.composition.runtime/1.3.0/System.Composition.Runtime.csproj"/>
+    <ProjectsToBuild Include="$(ReferenceAssemblySourcePath)system.composition.attributedmodel/1.3.0/System.Composition.AttributedModel.csproj"/>
+    <ProjectsToBuild Include="$(ReferenceAssemblySourcePath)system.composition.convention/1.3.0/System.Composition.Convention.csproj"/>
+    <ProjectsToBuild Include="$(ReferenceAssemblySourcePath)system.composition.hosting/1.3.0/System.Composition.Hosting.csproj"/>
+    <ProjectsToBuild Include="$(ReferenceAssemblySourcePath)system.composition.typedparts/1.3.0/System.Composition.TypedParts.csproj"/>
```

Comment:
The PR description lists 6 packages, but there are 5 here?

Reply:
I was just investigating that.  One didn't generate for some reason.  I'm going to remove it from the text and add it later if needed.